app/src/main/java/app/revanced/integrations/settings/SettingsEnum.java

```diff
@@ -179,6 +176,9 @@ public enum SettingsEnum {
             parents(SPOOF_SIGNATURE)),
     SPOOF_DEVICE_DIMENSIONS("revanced_spoof_device_dimensions", BOOLEAN, FALSE, true),
     BYPASS_URL_REDIRECTS("revanced_bypass_url_redirects", BOOLEAN, TRUE),
+    ANNOUNCEMENTS("revanced_announcements", BOOLEAN, TRUE, false),
+    ANNOUNCEMENT_CONSUMER("revanced_announcement_consumer", STRING, ""),
+    LAST_ANNOUNCEMENT_HASH("revanced_last_announcement_hash", STRING, ""),
```

Comment:
On second thought, the consumer is tied to the actual user, not just an installation. This should also be true for the RYD user id and the SponsorBlock local data. I have seen users accidentally lose them because they didn't assume that they would not be saved.

Reply:
RYD can be skipped in this case.
Regarding exporting, it would be better if there were a single export for everything related to ReVanced. This is because you usually plan to back up everything if you export or import. This is what many people expect, which is why many lose SB-specific data when they export their data because they assume that they already have backed up everything.

Because this is off-topic now, though, we can look into it sometime later.

> I can make a separate PR for that.

Sure